frontend/src/components/StarBallotView.js

```diff
@@ -30,119 +53,215 @@ const Choices = ({ rowIndex, onClick, ranking }) =>
   Array(6)
     .fill(0)
     .map((elem, n) => (
-      <td key={n} className="starScore">
+      <Grid item xs={1} align='center'>
         <Choice
           key={`starChoice${rowIndex}-${n}`}
           divKey={`starDiv${rowIndex}-${n}`}
           score={n}
           filled={n + 1 === ranking}
           onClick={() => onClick(n + 1)}
+
         />
-      </td>
+      </Grid>
     ));
 
 // Represents the row of all data for a single candidate
-const Row = ({ rowIndex, candidate, party, ranking, onClick }) => (
-  <tr key={`starRow${rowIndex}`}>
-    <RowHeading className="starScore" candidate={candidate} party={party} />
-    <Choices
-      key={`starChoices${rowIndex}`}
-      rowIndex={rowIndex}
-      ranking={ranking}
-      onClick={onClick}
-    />
-  </tr>
-);
+const Row = ({ rowIndex, candidate, party, ranking, onClick }) => {
+
+  const [expanded, setExpanded] = useState(false)
+  var rowColor = 'white'
+  if (rowIndex % 2 == 0) {
+    rowColor = '#f8f8f8';
+  } else {
+    rowColor = 'white';
+  }
+  return (
+    <>
+      <Grid container alignItems="center" style={{ backgroundColor: rowColor }}>
+        <Grid item xs={1}>
+          {!expanded &&
+            <IconButton aria-label="Home" onClick={() => { setExpanded(true) }}>
+              <ExpandMore />
+            </IconButton>}
+          {expanded &&
+            <IconButton aria-label="Home" onClick={() => { setExpanded(false) }}>
+              <ExpandLess />
+            </IconButton>}
+        </Grid>
+        <Grid item xs={4}>
+          <Typography align='left' variant="h6" component="h6">
+            {candidate.shortName}
+          </Typography>
+        </Grid>
+        <Choices
+          key={`starChoices${rowIndex}`}
+          rowIndex={rowIndex}
+          ranking={ranking}
+          onClick={onClick}
+        />
+
+      </Grid>
+      {expanded &&
+        <Grid container style={{ backgroundColor: rowColor }}>
+          <Grid item xs={1}>
+          </Grid>
+          <Grid item xs={5}>
+            <Typography align='left' s component="h6">
+              {candidate.fullName}
+            </Typography>
+            <Link color="inherit" href={candidate.candidateUrl} target="_blank">
```

Comment:
This might only work for relative links, I made some test elections and I couldn't open the links

Reply:
It seems to work if you copy and paste the links with the "https" otherwise it turns it into a relative link. I'll try to find a way around that.

Reply:
ah ok, I thought I did https the first time, but I tested it again and you're right